### PR TITLE
Add all ML models

### PR DESCRIPTION
**Pull request by ramaneswaran**

### CODE REVIEW

File: fashion_intel/imports.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# General libraries
+import os
+import json
+import random
+from ast import literal_eval
+from tqdm.notebook import tqdm as tqdm_notebook
```
[the remark this answers is not in the document]
Comment (ramaneswaran) [reply fashion_intel/imports.py:6]: tqdm_notebook is what is getting deprecated
the newer one is tqdm.notebook.tqdm

File: fashion_intel/apis/feed.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from fashion_intel.imports import *
+
+
```
[the remark this answers is not in the document]
Comment (ramaneswaran) [reply fashion_intel/apis/feed.py:3]: okay

File: fashion_intel/apis/reccomender.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from fashion_intel.imports import *
+
+
```
[the remark this answers is not in the document]
Comment (ramaneswaran) [reply fashion_intel/apis/reccomender.py:3]: okay

File: fashion_intel/fashion2vec/model.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+from fashion_intel.imports import *
+
```
[the remark this answers is not in the document]
Comment (ramaneswaran) [reply fashion_intel/fashion2vec/model.py:2]: okay

**Comment by oke-aditya** [comment]
Closed without Merging why @ramaneswaran

**Pull request closed without merging**